scripts/benchmark-rule.mjs

```diff
@@ -48,8 +47,7 @@ if (
 /* eslint-enable eqeqeq */
 const ruleSettings = normalizeRuleSettings(parsedOptions);
 
-const primary = ruleSettings[0];
-const secondary = ruleSettings[1] || null;
+const [primary, secondary] = ruleSettings;
```

Comment:
question

I suppose switching from `null` to `undefined` doesn't matter?

Reply:
Yes, it shouldn't have any problems. If something were wrong, the current validation logic would have problems.